charts/model-engine/templates/inference_framework_config.yaml

```diff
@@ -2,17 +2,18 @@ apiVersion: v1
 kind: ConfigMap
 metadata:
   name: {{ include "modelEngine.fullname" . }}-inference-framework-latest-config
+  namespace: {{ .Release.Namespace }}
   labels:
     product: common
     team: infra
   annotations:
-    "helm.sh/hook": pre-install
+    "helm.sh/hook": pre-install,pre-upgrade
```

Comment:
without this you can't update the vllm tags on upgrades

Reply:
ah, internally, we have a separate upgrade process for vllm that doesn't require install helm charts. If you're interested, we can socialize that a bit better

Reply:
oh rip i'll wrap this in a config then, it's easiest for use to change with upgrades but curious about your process